packages/text-annotator/src/state/spatialTree.ts

```diff
@@ -145,7 +149,7 @@ export const createSpatialTree = <T extends TextAnnotation>(store: Store<T>, con
 
   const getAnnotationRects = (id: string): DOMRect[] => {
     const indexed = index.get(id);
-    if (indexed) {
+    if (indexed[0]) {
```

Comment:
@oleksandr-danylchenko can you say why you made this change? I can see this might be useful as a guard. But wouldn't it break when querying for an ID that's not in the index?

Reply:
Yeah, it'll be breaking when asking for a non-existent id. But I didn't think about it as a straight disadvantage. 🤔 
Because the internal code that calls this method will only provide existing ids. And for the consumers that wanna call it from the outside, we set an implicit contract - you must provide an adequate and existing ID for use to return you a value.

Reply:
Hm, if people _can_ call it from outside, they'll call it with all sorts of things. I'd rather protect against that. Is there a case for an annotation being in the index with no rects? Technically, that's currently possible. But I wonder if that shouldn't be filtered out perhaps?

Reply:
> Is there a case for an annotation being in the index with no rects?

IIRC, I faced that when we used the "selection presence" check to open the popup. On the 1st call of the `getBoundingClientRect` the annotation was in the index but didn't have rects in the spatial tree. But that was quite a while ago 🤔

Reply:
See https://github.com/recogito/text-annotator-js/commit/efa4b06ed87dc595f8ac31514cd45bc16452cd8b

Reply:
Oh, right... That looks sensible, thanks 🙌🏻